pvlib/pvsystem.py

```diff
@@ -2230,12 +2230,12 @@ def sapm(effective_irradiance, temp_cell, module):
 
     Notes
     -----
-    The SAPM parameters which are required in ``module`` are
-    listed in the following table.
-
     The Sandia module database contains parameter values for a limited set
     of modules. The CEC module database does not contain these parameters.
-    Both databases can be accessed using :py:func:`retrieve_sam`.
+    Both databases can be accessed using :py:func:`retrieve_sam`. The full list
+    of SAPM parameters is presented in the table below. All but ``A0`` to
+    ``A4``, ``B0`` to ``B5``, ``C4`` to ``C7``, ``IX0``, and ``IXX0`` are
```

Comment:
```suggestion
    ``A4``, ``B0`` to ``B5``, ``C4`` to ``C7``, ``IXO``, and ``IXXO`` are
```

Reply:
I would prefer to rename the parameters to `IX0` and `IXX0` but let's not do that.